docs/control-structures.rst

```diff
@@ -742,7 +742,7 @@ A failure in an external call can be caught using a try/catch statement, as foll
 ::
 
     // SPDX-License-Identifier: GPL-3.0
-    pragma solidity >=0.6.0 <0.9.0;
+    pragma solidity ^0.8.1;
```

Comment:
Maybe will fix failure:
```suggestion
    pragma solidity >0.8.0;
```

Reply:
I think we should fix the script: If the release is not available, use the current branch or something like that.

Reply:
I see you added a workaround specifically to  be able to use non-existent `.99` versions: `b03d1105289eb3fb53dfbf58916aed56a13cf3e6` :)

So either use that or we could just remove the `exit 1` from the script and it would just skip these errors.

If you want, we could also make it ignore version one higher than the latest release - it does download `list.txt` so we have all the necessary information to determine that.

Reply:
We cannot use `.99` here. I want to use the next non-breaking release that is not released yet.

Reply:
We could just add a special case that assumes that the version of the current branch / the develop branch has been released.

Reply:
Oh, right. I didn't look at the example closely.

Well, sure, a special case for version from develop sounds pretty easy. I can do that. I'll submit a PR tomorrow.

Reply:
My PR that solves the problem:  #10770. You can now rebase your PR on it and see if it works.